### PR TITLE
Use epoch nanos everywhere instead of nano time in the Span implementation.

### PR DESCRIPTION
**Pull request by bogdandrutu**
Fixes https://github.com/open-telemetry/opentelemetry-java/issues/633

### CODE REVIEW

File: sdk/src/main/java/io/opentelemetry/sdk/trace/RecordEventsReadableSpan.java [hunk 1]
```diff
@@ -429,23 +419,23 @@ public void updateName(String name) {
 
   @Override
   public void end() {
-    endInternal();
+    endInternal(clock.now());
   }
 
   // TODO: Use endOptions.
```
Comment (carlosalberto) [inline sdk/src/main/java/io/opentelemetry/sdk/trace/RecordEventsReadableSpan.java]: Remove this line?

Comment (carlosalberto) [reply sdk/src/main/java/io/opentelemetry/sdk/trace/RecordEventsReadableSpan.java]: (And likewise for the rest ;) )

Comment (bogdandrutu) [reply sdk/src/main/java/io/opentelemetry/sdk/trace/RecordEventsReadableSpan.java]: Done.

**Comment by bogdandrutu** [comment]
@/all please take another look

File: sdk/src/main/java/io/opentelemetry/sdk/internal/MonotonicClock.java [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+    long deltaNanos = clock.nanoTime() - this.nanoTime;
+    return epochNanos + deltaNanos;
+  }
+
+  @Override
+  public long nanoTime() {
+    return clock.nanoTime();
+  }
+
+  private MonotonicClock(Clock clock, long epochNanos, long nanoTime) {
```
Comment (jkwatson) [inline sdk/src/main/java/io/opentelemetry/sdk/internal/MonotonicClock.java]: can you move the constructor up above the methods?

Comment (bogdandrutu) [reply sdk/src/main/java/io/opentelemetry/sdk/internal/MonotonicClock.java]: You mean as the first method?

Comment (bogdandrutu) [reply sdk/src/main/java/io/opentelemetry/sdk/internal/MonotonicClock.java]: Done.

Comment (jkwatson) [reply sdk/src/main/java/io/opentelemetry/sdk/internal/MonotonicClock.java]: 🐎

**Pull request merged**